templates/zerver/help/see-all-images-in-a-narrow.md

```diff
@@ -0,0 +1,14 @@
+# See all images in a narrow
+
+When someone sends an image in a message, Zulip generates a small preview if
+the image is a common image type, like PNG, JPEG, or GIF.
+
+To **view the image at full size**, click on the preview. From there, you
+can **pan and zoom**, **download** the image, and **navigate to other
+images** in the narrow. You can close the full screen view by clicking
+anywhere outside the image.
+
+!!! tip ""
```

Comment:
We have a new keyboard_tip for keyboard shortcuts. see templates/zerver/help/send-a-private-message.md